App.js

```diff
@@ -1,18 +1,173 @@
 import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import { Alert, AppRegistry, Button, StyleSheet, Text, TextInput, View } from 'react-native';
+import { StackNavigator } from 'react-navigation';
+import Picker from 'react-native-picker';
+import { Dropdown } from 'react-native-material-dropdown';
+
+// export default class LogIn extends React.Component {
+//   // static navigationOptions = {
+//   //   title: 'Sign Up',
+//   // };
+//   constructor(props) {
+//     super(props);
```

Comment:
It currently isn't doing anything, I'm pretty sure it won't be doing anything in the future either so it'll likely be gone soon.